component.html

```diff
@@ -134,6 +135,146 @@ <h3 class="avatar-heading">Alert</h3>
+      </p>
+
+      <div class="badge-items flex">
+        <div class="badge-avatar flex  badge-xl">
+          <img src="./Asset/kitty.png" alt="" class="kitty avatar-xl" /><span
+            class="icon-avatar-badge flex icon-l"
+          ></span>
+        </div>
+        <div class="badge-avatar flex badge-xl">
+          <img src="./Asset/kitty.png" alt="" class="kitty avatar-lg" /><span
+            class="icon-avatar-badge flex icon-l"
+          ></span>
+        </div>
+        <div class="badge-avatar flex badge-xl">
+          <img src="./Asset/kitty.png" alt="" class="kitty avatar-md" /><span
+            class="icon-avatar-badge flex icon-l"
+          ></span>
+        </div>
+      </div>
+      <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28248%2C192%2C200%2C0.94%29&t=base16-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=39px&ph=50px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=13.5px&lh=130%25&si=false&es=2x&wm=false&code=%250A%253Cdiv%2520class%253D%2522badge-avatar%2520badge-xl%2522%253E%250A%2520%2520%2520%253Cimg%2520src%253D%2522.%252FAsset%252Fkitty.png%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520avatar-xl%2522%2520%252F%253E%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522icon-avatar-badge%2520icon-l%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522badge-avatar%2520badge-xl%2522%253E%250A%2520%2520%2520%253Cimg%2520src%253D%2522.%252FAsset%252Fkitty.png%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520avatar-lg%2522%2520%252F%253E%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522icon-avatar-badge%2520icon-l%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522badge-avatar%2520badge-xl%2522%253E%250A%2520%2520%2520%253Cimg%2520src%253D%2522.%252FAsset%252Fkitty.png%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520avatar-md%2522%2520%252F%253E%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522icon-avatar-badge%2520icon-l%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%2520%2520%250A"
+  style="width: 100%; height: 398px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+    </section>
+
+
+    <!-- --- Button ---- -->
+
+    <section class="main-title" id="buttons">
+      <h2 class="avatar-heading">Button</h2>
+      <p class="component-navbar-item">Buttons are also called as call to action. We have range of buttons and their states. You may use a tag or button element, you need to add respective classes, and you are good to go.</p>
+      <h3 class="avatar-heading">Primary Button</h3>
+      <p class="component-navbar-item"> 
+        Whenever you want your user to click on a link or button, use primary style buttons.  
+      </p>
+
+      <div class="button-items">
+        <p>
+          <button class="btn btn-primary btn-round-1">Primary</button>
```

Comment:
I think there no other btn-round classes so instead of btn-round-1 you can use btn-round

Reply:
Done